test/lifecycle_test.dart

```diff
@@ -368,6 +420,7 @@ void sharedLifecycleTests<T extends react.Component>({
 
       const Map expectedState = const {};
       const Map expectedContext = const {};
+      const Null expectedSnapshot = null;
```

Comment:
well these are pretty hardcore for a null check lol @greglittlefield-wf is this needed or could he just do: `var expectedSnapshot;`

Reply:
This looks good to me; it should definitely be a `const` as opposed to a var